tests/unit/lib/Any_unittest.cc

```diff
@@ -693,10 +693,10 @@ TEST(Any, lazy_cloneable_visitable)
 	    any.clone_from(make_any<int>([=]() { return 111; })), std::logic_error);
 	EXPECT_TRUE(any.cloneable());
 	EXPECT_TRUE(any.visitable());
-	any = make_any<int>([=]() { return 222; });
-	EXPECT_FALSE(any.cloneable());
-	EXPECT_FALSE(any.visitable());
-	EXPECT_THROW(any.visit(nullptr), std::logic_error);
```

Comment:
🤔 is this why is WIP?

Reply:
yea, it was also failing until now. Now all tests pass at least in my machine.. 
Need to figure out what can be visitable.. With this patch it's everything except void functions